salt/modules/yumpkg.py

```diff
@@ -147,6 +147,9 @@ def _yum():
         if ('fedora' in __grains__['os'].lower()
            and int(__grains__['osrelease']) >= 22):
             __context__[contextkey] = 'dnf'
+        elif ('centos' in __grains__['os'].lower()
+           and int(__grains__['osrelease'].split('.')[0]) > 7):
```

Comment:
greater than matches how it's done with Fedora.

```suggestion
           and int(__grains__['osrelease'].split('.')[0]) >= 8):
```